mex/common/identity/memory.py

```diff
@@ -46,11 +48,23 @@ def assign(
         if identities:
             return identities[0]
 
+        def get_identifier(target_field: str) -> Identifier:
```

Comment:
```suggestion
    @staticmethod
    def _get_identifier(target_field: str) -> Identifier:
```

i don't like closures, as they discourage unit tests

Reply:
refactored into staticmethod in e088314

Reply:
much obliged 🙏